### ISSUE TITLE
Additional Resources

### ISSUE THREAD

**Issue by bradfrost** [bradfrost/frontend-guidelines-questionnaire#6]
- http://taitems.github.io/Front-End-Development-Guidelines/
- http://tao.wearefriday.com/engineering/frontend/

Anybody have others?

**Comment by rodrigoibarra** [comment]
CSS guidelines by Harry perhaps?

http://cssguidelin.es/

**Comment by thibaudcolas** [comment]
This isn't specifically guidelines nor front-end, but another interesting example of "defining common practices to improve consistency and cohesion": https://ways-of-working.readthedocs.org/

**Comment by nathanacurtis** [comment]
The "examples" position the conventional answers of each question, yet I wonder:
- Could some questions have longer tails of alternatives?
- Could you standardize the display of those conventional choices (as opposed to the inconsistent parenthetical references)?

Regardless, a few to consider adding:
- Templating: [Nunjucks](https://mozilla.github.io/nunjucks/)
- CSS Preprocessor: [Stylus](http://stylus-lang.com/)
- CSS Grids (often a subsystem many hijack, distinguished from adopting the whole framework): [Jeet](http://jeet.gs/), [Susy](http://susy.oddbird.net/), [Pure](http://purecss.io/grids/) or Bootstrap
- Browser syncing
- Browser testing like [Browserstack](https://www.browserstack.com/)

**Comment by bradfrost** [comment]
@nathanacurtis,

> Could some questions have longer tails of alternatives?

Yes, although it very much risks a slippery slope of having to maintain a comprehensive list of tools and resources. Been there, done that and I'd prefer this stays a suggestive rather than exhaustive list. (That being said, it would be cool to see a more interactive version of this that had more of a comprehensive list!)

> Could you standardize the display of those conventional choices (as opposed to the inconsistent parenthetical references)?

I can visualize an interactive step-by-step questionnaire that would be in essence a form for answering these questions. That experience could have a standardized display.

**Comment by nathanacurtis** [comment]
👍🏻

On Saturday, January 16, 2016, Brad Frost notifications@github.com wrote:

> @nathanacurtis https://github.com/nathanacurtis,
> 
> Could some questions have longer tails of alternatives?
> 
> Yes, although it very much risks a slippery slope of having to maintain a
> comprehensive list of tools and resources. Been there, done that and I'd
> prefer this stays a suggestive rather than exhaustive list. (That being
> said, it would be cool to see a more interactive version of this that had
> more of a comprehensive list!)
> 
> Could you standardize the display of those conventional choices (as
> opposed to the inconsistent parenthetical references)?
> 
> I can visualize an interactive step-by-step questionnaire that would be in
> essence a form for answering these questions. That experience could have a
> standardized display.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/bradfrost/frontend-guidelines-questionnaire/issues/6#issuecomment-172260056
> .

## 

_Nathan Curtis_
Principal, EightShapes LLC
nathan@eightshapes.com
@nathanacurtis
703-296-8831
